src/components/ResultsCard/ResultsCard.tsx

```diff
@@ -88,14 +89,17 @@ function renderOriginal(item, alphabets, index) {
                         lang={lang}
                     />
                 );
-            })} {item.ipa && <span className="ipa">[{item.ipa}]</span>}
+            })} 
+            {item.caseInfo && <> <span className="caseInfo">(+{t(`case${item.caseInfo.slice(2,-1)}`)})</span></>}
```

Comment:
Maybe it's worth to add a separate small render function for caseInfo?

Reply:
I don't want to make a function here yet...